### DISCUSSION TITLE
how to get online players

### DISCUSSION THREAD

**Discussion by ntloff** [PrismarineJS/mineflayer#1495] [Q&A] [answered]
how to get online players on de srver?

**Comment by ghost** [comment, marked as answer]
use
```js
bot.players
```
Please fill out the template

**Reply by DereC4** [reply]
ty

**Comment by ghost** [comment]
> how to get online players on de srver?

Read the documentation and there will be no such questions

**Comment by ntloff** [comment]
> ```js
> const playerList = Object.keys(bot.players).join(", ")
> 
> console.log(playerList)
> ```
> 
> You can get faster replies in the Discord

D:\!minecraft\minecorp\minecorp.js:12
const playerList = Object.keys(bot.players).join(", ");
                               ^

ReferenceError: Cannot access 'bot' before initialization
    at Object.<anonymous> (D:\!minecraft\minecorp\minecorp.js:12:32)
    at Module._compile (internal/modules/cjs/loader.js:1138:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10)
    at Module.load (internal/modules/cjs/loader.js:986:32)
    at Function.Module._load (internal/modules/cjs/loader.js:879:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at internal/main/run_main_module.js:17:47

**Comment by ntloff** [comment]
how to set topic via plaers online

**Comment by ntloff** [comment]
channel.setTopic("hi") is not working

**Reply by moderncraft1234** [reply]
channel.setTopic(`hi`)

**Reply by satyamedh** [reply]
why did I get an email for this? GITHUB?!??!?!

**Comment by ghost** [comment]
> channel.setTopic("hi") is not working
> 

Please show the code for better help and understanding of the situation
